dangerfile.js

```diff
@@ -60,11 +60,16 @@ Sentry runs a symbolicator integration test suite located at [\`tests/symbolicat
 Follow these steps to update snapshots in Sentry:
 
 1. Check out latest Sentry \`master\` and enable the virtualenv.
-2. Stop the symbolicator devservice using \`sentry devservices down symbolicator\`.
-3. Run your development symbolicator on port \`3021\`.
-4. Export \`SENTRY_SNAPSHOTS_WRITEBACK=1\` and run symbolicator tests with pytest.
-5. Review snapshot changes locally, then create a PR to Sentry.
-6. Merge the Symbolicator PR, then merge the Sentry PR.
+2. Make sure your other devservices are running via \`sentry devservices up\`, and that symbolicator
+is enabled (\`symbolicator: true\`) in sentry via \`~/.sentry/config.yml\`.
+4. Stop the symbolicator devservice using \`sentry devservices down symbolicator\`. You want to use your
```

Comment:
I think [you can use `--exclude`](https://github.com/getsentry/sentry/blob/d15e14f01c597353bc40972043a83df11fa99999/src/sentry/runner/commands/devservices.py#L153) to start devservices w/o symbolicator